src/model/AbstractEntity.ts

```diff
@@ -0,0 +1,4 @@
+/**
+ * Entities represent how the data is stored in the database.
+ */
+export interface AbstractEntity extends Record<string, any> {}
```

Comment:
BaseEntity or IEntity is a better name. Abstract keyword may imply there's a base class or something of the sort. Also, I would use the Record Type directly without the AnbstractEntity, because the latter is just another layer of abstraction that serves no purpose.

Reply:
@zubietaroberto Make this modification and I think we're good to go

Reply:
@jeancaldron this interface was already deleted.